packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -181,32 +192,32 @@ export const addApiConfig = () => {
       /^(\s*)(schema: makeMergedSchema)(.*)$/m,
       `$1getCurrentUser,\n$1$2$3`
     )
-    fs.writeFileSync(API_GRAPHQL_PATH, content)
+    fs.writeFileSync(graphqlPath, content)
   }
 }
 
-export const isProviderSupported = (provider) => {
-  return SUPPORTED_PROVIDERS.indexOf(provider) !== -1
+export const isProviderSupported = (provider, supportedProviders) => {
+  return supportedProviders.indexOf(provider) !== -1
```

Comment:
Why can't you call `getSupportedProviders()` here? Instead of passing it in as a parameter

Reply:
good remark, I thought in case if it is needed to check more than once to not calls the method many times, but in this context, is always only one provider per run.